packages/babel-helper-create-class-features-plugin/src/fields.js

```diff
@@ -224,11 +224,20 @@ const privateNameHandlerSpec = {
     }
 
     if (isMethod) {
-      if (isAccessor) {
+      if (getId) {
         return t.callExpression(file.addHelper("classPrivateFieldGet"), [
           this.receiver(member),
           t.cloneNode(id),
         ]);
+      } else if (setId) {
+        if (file.availableHelper("writeOnlyError")) {
+          return t.callExpression(file.addHelper("writeOnlyError"), [
+            t.stringLiteral(name),
+          ]);
+        }
+        console.warn(
+          `@babel/helpers is outdated, update it to silence this warning.`,
+        );
```

Comment:
If the `writeOnlyError` helper is not available, we should still transform using
``` t.callExpression(file.addHelper("classPrivateFieldGet"), [
          this.receiver(member),
          t.cloneNode(id),
        ]);
```
otherwise it becomes a breaking change for those update `@babel/deps` but pin `@babel/helper`.

Reply:
Right🤦🏻‍♂️